### ISSUE TITLE
Documentation generation failed on mono

### ISSUE THREAD

**Issue by gsvgit** [tpetricek/FSharp.Formatting#414]
I have a problem with documentation generation on mono. On Win \ .NET all works fine.

My paket.lock:
```
GROUP Build
NUGET
  remote: https://www.nuget.org/api/v2
    FAKE (4.30.2)
    FSharp.Compiler.Service (2.0.0.6)
    FSharp.Formatting (2.14.4)
      FSharp.Compiler.Service (2.0.0.6)
      FSharpVSPowerTools.Core (>= 2.3 < 2.4)
    FSharpVSPowerTools.Core (2.3)
      FSharp.Compiler.Service (>= 2.0.0.3)
    Microsoft.Bcl (1.1.10) - framework: net10, net11, net20, net30, net35, net40, net40-full
      Microsoft.Bcl.Build (>= 1.0.14)
    Microsoft.Bcl.Build (1.0.21) - import_targets: false, framework: net10, net11, net20, net30, net35, net40, net40-full
    Microsoft.Net.Http (2.2.29) - framework: net10, net11, net20, net30, net35, net40, net40-full
      Microsoft.Bcl (>= 1.1.10)
      Microsoft.Bcl.Build (>= 1.0.14)
    Octokit (0.20)
      Microsoft.Net.Http  - framework: net10, net11, net20, net30, net35, net40, net40-full
    SourceLink.Fake (1.1)

```
Stack trace:
```
Error: System.Exception: Generating HTML failed.
	  at FSharp.Literate.RazorRender.handleCompile[a] (System.String source, Microsoft.FSharp.Core.FSharpFunc`2 f) <0x427f7120 + 0x00738> in <filename unknown>:0 
	  at FSharp.Literate.RazorRender.ProcessFileModel[b] (System.Type modelType, System.Object model, Microsoft.FSharp.Core.FSharpOption`1 properties) <0x427f6f00 + 0x0009b> in <filename unknown>:0 
	  at FSharp.Literate.Templating.generateFile (Microsoft.FSharp.Core.FSharpOption`1 references, System.String contentTag, IEnumerable`1 parameters, Microsoft.FSharp.Core.FSharpOption`1 templateOpt, System.String output, IEnumerable`1 layoutRoots) <0x427f2350 + 0x001f7> in <filename unknown>:0 
	  at FSharp.Literate.Templating.processFile (Microsoft.FSharp.Core.FSharpOption`1 references, FSharp.Literate.LiterateDocument doc, System.String output, FSharp.Literate.ProcessingContext ctx) <0x427b2000 + 0x004a6> in <filename unknown>:0 
	  at FSharp.Literate.Literate.ProcessScriptFile (System.String input, Microsoft.FSharp.Core.FSharpOption`1 templateFile, Microsoft.FSharp.Core.FSharpOption`1 output, Microsoft.FSharp.Core.FSharpOption`1 format, Microsoft.FSharp.Core.FSharpOption`1 formatAgent, Microsoft.FSharp.Core.FSharpOption`1 prefix, Microsoft.FSharp.Core.FSharpOption`1 compilerOptions, Microsoft.FSharp.Core.FSharpOption`1 lineNumbers, Microsoft.FSharp.Core.FSharpOption`1 references, Microsoft.FSharp.Core.FSharpOption`1 fsiEvaluator, Microsoft.FSharp.Core.FSharpOption`1 replacements, Microsoft.FSharp.Core.FSharpOption`1 includeSource, Microsoft.FSharp.Core.FSharpOption`1 layoutRoots, Microsoft.FSharp.Core.FSharpOption`1 generateAnchors, Microsoft.FSharp.Core.FSharpOption`1 assemblyReferences, Microsoft.FSharp.Core.FSharpOption`1 customizeDocument) <0x42464c20 + 0x000df> in <filename unknown>:0 
	  at <StartupCode$FSharp-Literate>.$Main+processScriptFile@184.Invoke (System.String file, System.String output) <0x42464ac0 + 0x0012d> in <filename unknown>:0 
	  at Microsoft.FSharp.Core.FSharpFunc`2[T,TResult].InvokeFast[V] (Microsoft.FSharp.Core.FSharpFunc`2 func, Microsoft.FSharp.Core.T arg1, Microsoft.FSharp.Core.TResult arg2) <0x41ead570 + 0x0008f> in <filename unknown>:0 
	  at <StartupCode$FSharp-Literate>.$Main.processDirectory@199 (Microsoft.FSharp.Core.FSharpOption`1 format, Boolean processRecursive, Microsoft.FSharp.Core.FSharpFunc`2 processScriptFile, Microsoft.FSharp.Core.FSharpFunc`2 processMarkdown, System.String indir, System.String outdir) <0x42462820 + 0x00922> in <filename unknown>:0 
	  at FSharp.Literate.Literate.ProcessDirectory (System.String inputDirectory, Microsoft.FSharp.Core.FSharpOption`1 templateFile, Microsoft.FSharp.Core.FSharpOption`1 outputDirectory, Microsoft.FSharp.Core.FSharpOption`1 format, Microsoft.FSharp.Core.FSharpOption`1 formatAgent, Microsoft.FSharp.Core.FSharpOption`1 prefix, Microsoft.FSharp.Core.FSharpOption`1 compilerOptions, Microsoft.FSharp.Core.FSharpOption`1 lineNumbers, Microsoft.FSharp.Core.FSharpOption`1 references, Microsoft.FSharp.Core.FSharpOption`1 fsiEvaluator, Microsoft.FSharp.Core.FSharpOption`1 replacements, Microsoft.FSharp.Core.FSharpOption`1 includeSource, Microsoft.FSharp.Core.FSharpOption`1 layoutRoots, Microsoft.FSharp.Core.FSharpOption`1 generateAnchors, Microsoft.FSharp.Core.FSharpOption`1 assemblyReferences, Microsoft.FSharp.Core.FSharpOption`1 processRecursive, Microsoft.FSharp.Core.FSharpOption`1 customizeDocument) <0x42461df0 + 0x00238> in <filename unknown>:0 
	  at FSI_0005.Generate.buildDocumentation () <0x42461410 + 0x0033c> in <filename unknown>:0 
	  at <StartupCode$FSI_0005>.$FSI_0005_Generate$fsx.main@ () <0x4245b5f0 + 0x00d71> in <filename unknown>:0 
	  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&)
	  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) <0x7fa90a35a5d0 + 0x000a1> in <filename unknown>:0 
	Stopped due to error
```
Full log: 
https://travis-ci.org/YaccConstructor/Brahma.FSharp/builds/144752901

**Comment by matthid** [comment]
The fix is: Don't use mono 4.4 see
- https://github.com/fsprojects/ProjectScaffold/issues/252
- https://github.com/fsharp/FAKE/issues/1196
- https://bugzilla.xamarin.com/show_bug.cgi?id=41979

**Comment by gsvgit** [comment]
Thanks!
